### DISCUSSION TITLE
KAPE Download Error

### DISCUSSION THREAD

**Discussion by josephkilatya** [EricZimmerman/KapeFiles#808] [Q&A] [answered]
I am facing problems while trying to download the tool. I keep getting the error message to try again later. How do I solve this issue?

**Comment by EricZimmerman** [comment]
did you try a different computer? disable any adblocking, etc?

**Reply by josephkilatya** [reply]
My bad I hadn`t disabled the adblocker. It worked out. Thank you

**Comment by AndrewRathbun** [comment, marked as answer]
Try without ad blockers or other extensions that make the internet generally more pleasant to use. Or try incognito. That should allow you to get the download request form successfully submitted.

**Reply by josephkilatya** [reply]
It worked after disabling the adblocker. Thank you.

**Reply by AndrewRathbun** [reply]
> It worked after disabling the adblocker. Thank you.

Glad to hear. Cheers!

**Reply by zorone** [reply]
Sometimes just using incognito might also not work. Please double check your extension, you might allow some extensions to run in incognito mode.
